app/controllers/invite_requests_controller.rb

```diff
@@ -63,6 +67,8 @@ def reorder
 
   def destroy
     @invite_request = InviteRequest.find_by(id: params[:id])
+    authorize @invite_request || authorize(InviteRequest)
```

Comment:
Do we want to switch from `find_by` to `find` in the line above, so we get a 404 when the request can't be found? Otherwise the `find_by` means `@invite_request` can be `nil` and we either need to do `authorize(InviteRequest)` or `authorize @invite_request || authorize(InviteRequest)` to handle cases where the ID passed is for a nonexistent request.

Reply:
...okay, well, I didn't mean to commit that, but I did, so I guess I'll just finish it.